### PR TITLE
refactor(ui): fix small issues around navigation

### PR DESCRIPTION
**Pull request by rudolfs**
Small improvements that are necessary for https://github.com/radicle-dev/radicle-upstream/pull/1813.
See commit messages for more details.

### CODE REVIEW

File: ui/Screen/Project/ProjectHeader.svelte [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  </h1>
+  <RadicleId truncate {urn} style="margin-top: 0.5rem;" />
+  {#if description.length > 0}
+    <p class="description typo-overflow-ellipsis" title={description}>
+      {description}
+    </p>
+  {/if}
+  {#if stats}
+    <div class="stats" data-cy="project-stats">
+      {#if stats && stats.branches > 0}
```
Comment (geigerzaehler) [inline ui/Screen/Project/ProjectHeader.svelte:73]: Can you check if we need to check for `stats` or if the following is enough?
```suggestion
      {#if stats.branches > 0}
```
Same also goes for the two if-statements below

File: ui/Screen/Profile/ProfileHeader.svelte [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<script lang="typescript">
+  import type { Avatar as AvatarT } from "../../src/proxy/identity";
+
+  import { Avatar } from "../../DesignSystem/Primitive";
+  import { PeerId } from "../../DesignSystem/Component";
+
+  export let name: string;
+  export let peerId: string;
+
+  export let avatarFallback: AvatarT | undefined = undefined;
```
Comment (geigerzaehler) [inline ui/Screen/Profile/ProfileHeader.svelte:10]: Can you check if this property is always provided and whether we can remove the `undefined` case?

File: ui/Screen/UserProfile/UserProfileHeader.svelte [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<script lang="typescript">
+  import type { Avatar as AvatarT } from "../../src/proxy/identity";
+
+  import { Avatar } from "../../DesignSystem/Primitive";
+  import { PeerId } from "../../DesignSystem/Component";
+
+  export let name: string;
+  export let peerId: string;
+
+  export let avatarFallback: AvatarT | undefined = undefined;
```
Comment (geigerzaehler) [inline ui/Screen/UserProfile/UserProfileHeader.svelte:10]: Can you check if this property is always provided and whether we can remove the `undefined` case?

File: ui/Screen/Profile/ProfileHeader.svelte (hunk 2 above)
Comment (rudolfs) [reply ui/Screen/Profile/ProfileHeader.svelte]: As far as I can see, Identity always has the `avatarFallback`: 738f4194332d1f45366883dd2fd42cff81d15eed.

File: ui/Screen/UserProfile/UserProfileHeader.svelte (hunk 3 above)
Comment (rudolfs) [reply ui/Screen/UserProfile/UserProfileHeader.svelte]: As far as I can see, Identity always has the `avatarFallback`: 738f4194332d1f45366883dd2fd42cff81d15eed.

File: ui/Screen/Project/ProjectHeader.svelte (hunk 1 above)
Comment (rudolfs) [reply ui/Screen/Project/ProjectHeader.svelte]: c6987c12f30e2263784da02d39c706d59bda3ff9

**Review by geigerzaehler — DISMISSED** [review]

**Pull request merged**